tests/test_packages/test_contracts/test_erc1155/test_contract.py

```diff
@@ -693,11 +692,6 @@ def finish_contract_deployment(cls) -> str:
         if contract_address is None:
             raise ValueError("Contract address not found!")  # pragma: nocover
 
-        if contract_address != cast(
-            FetchAIApi, cls.ledger_api
-        ).get_last_contract_address(code_id):
-            raise ValueError("Contract address not valid!")  # pragma: nocover
-
```

Comment:
Explain the removals of `get_last_contract_address` and `get_last_code_id` similar to above.